zebra-state/src/service/finalized_state/zebra_db/block.rs

```diff
@@ -456,45 +455,49 @@ impl DiskWriteBatch {
         prev_note_commitment_trees: Option<NoteCommitmentTrees>,
     ) -> Result<(), BoxError> {
         let db = &zebra_db.db;
-        // Commit block and transaction data.
-        // (Transaction indexes, note commitments, and UTXOs are committed later.)
+        // Commit block, transaction, and note commitment tree data.
         self.prepare_block_header_and_transaction_data_batch(db, &finalized.verified)?;
 
+        // The consensus rules are silent on shielded transactions in the genesis block,
+        // because there aren't any in the mainnet or testnet genesis blocks.
+        // So this means the genesis anchor is the same as the empty anchor,
+        // which is already present from height 1 to the first shielded transaction.
+        //
+        // In Zebra we include their nullifiers and note commitments because it simplifies our code.
```

Comment:
Without reading the code below, it wasn't clear to me what "their" refers to. I suggest:

```suggestion
        // In Zebra we include the nullifiers and note commitments in the genesis block because it simplifies our code.
```